Project.toml

```diff
@@ -19,6 +18,27 @@ SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 TerminalLoggers = "5d786b92-1e48-4d6f-9151-6b4477ca9bed"
 
+[weakdeps]
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
+FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
+ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
+Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
+
+[extensions]
+EnzymeExt = "Enzyme"
+FinitediffExt = "FiniteDiff"
+ForwarddiffExt = "ForwardDiff"
+MTKExt = "ModelingToolkit"
+ReversediffExt = "ReverseDiff"
+TrackerExt = "Tracker"
+ZygoteExt = "Zygote"
```

Comment:
Please follow the recommended naming scheme, otherwise this will cause breakage.

Reply:
Do you mean camel-case for the `Diff`s? Otherwise it matches the recommended patter iiuc

Reply:
No it's missing the Optimization part. See other libraries like DiffEqBase, ArrayInterface, ...